addons/fools/index.js

```diff
@@ -0,0 +1,277 @@
+export default async function ({ addon, global, console }) {
+  document.addEventListener("keydown", async (e) => {
+    if (e.key == "s") {
+      await setup();
+    }
+  });
+  async function setup() {
+    await wait(300);
+    document.body.style.overflow = "hidden";
+    // create full screen canvas
+    const canvas = document.createElement("canvas");
+    canvas.width = window.innerWidth;
+    canvas.height = window.innerHeight;
+    canvas.style.position = "fixed";
+    canvas.style.top = "0";
+    canvas.style.left = "0";
+    canvas.style.zIndex = "1000";
+    document.body.appendChild(canvas);
+    canvas.style.transition = "opacity 3s ease-in-out";
+    canvas.style.opacity = "0.1";
+
+    canvas.style.backgroundColor = "black";
```

Comment:
Just going on the homepage now there are some thumbnails with pure black backgrounds.
```suggestion
    canvas.style.backgroundColor = "#111";
```